lib/methodfinder.rb

```diff
@@ -48,12 +48,17 @@ module MethodFinder
     CLASS_METHOD_BLACKLIST[:Object] << :pry
   end
 
-  @debug = ENV['METHOD_FINDER_DEBUG']
+  # true if METHOD_FINDER_DEBUG is truthy, false otherwise e.g.:
+  #
+  #   $ METHOD_FINDER_DEBUG=1 irb # true
+  #   $ METHOD_FINDER_DEBUG=0 irb # false
+  #   $ METHOD_FINDER_DEBUG=  irb # false
+  @debug = ![nil, '0', ''].include?(ENV['METHOD_FINDER_DEBUG'])
```

Comment:
Based on previous experience I can tell you that people will expect `false` to work too (and it should probably not be case-sensitive either).